samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
    :local:
    :depth: 2
 
-This bootloader sample implements an immutable first stage bootloader that has the capability to update the application firmware on the network core of the nRF5340 System on Chip (SoC).
-In addition to this, the network core bootloader sample performs flash protection of both itself and the application.
+The network core bootloader sample implements an immutable first-stage bootloader that can update the application firmware on the network core of the nRF5340 System on Chip (SoC).
 
 Overview
 ********
 
-The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+This sample supports transporting an already verified and authenticated firmware update from the application core flash memory to the network core flash memory.
+Also, this sample protects the flash memory areas allocated to both itself and the application running on the network core.
 
-This is accomplished by the following steps:
+The bootloader accomplishes these objectives as follows:
 
-1. Lock the flash of the bootloader.
-     The bootloader sample locks the flash that contains the sample bootloader and its configuration.
+1. Locking the flash memory areas containing the bootloader.
+     The bootloader sample locks the flash memory areas that contain the sample bootloader and its configuration.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
 
-#. Perform any pending network core firmware upgrade.
-     Invoke the :ref:`subsys_pcd` library to inspect a SRAM region shared with the application core.
-     If the application core has written an update instruction, copy the specified data range to the application partition on the network core.
-     Once the copy is done, compare the SHA of the data in the application partition against the SHA specified in the shared SRAM.
-     Communicate the result of the comparison to the application core through the shared SRAM.
+#. Performing any pending network core firmware update.
+     The sample calls the :ref:`subsys_pcd` library to inspect the SRAM region shared with the application core:
 
-#. Lock the flash of the application.
-     Lock the flash that contains the application.
+     a. If the application core has written an update instruction, the bootloader copies the specified data range to the application partition on the network core.
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
+     #. It then communicates the result of the comparison to the application core through the shared SRAM.
+
+#. Locking the flash memory areas containing the application.
+     The sample locks the flash memory areas that contain the application.
```

Comment:
This sentence is exactly the same as the heading - no need for duplication. If you have more information to add, you should extend the second sentence; otherwise just get rid of it.
If you want to be parallel to the first step, you could extend the heading for the first step ("... containing the bootloader and its configuration") and get rid of the duplication there as well.

Reply:
fair enought, fixed